tgui/packages/tgui/interfaces/CryopodConsole.js

```diff
@@ -0,0 +1,91 @@
+import { useBackend } from '../backend';
+import { Button, Section, NoticeBox, LabeledList, Collapsible } from '../components';
+import { Window } from '../layouts';
+
+export const CryopodConsole = (props, context) => {
+  const { data } = useBackend(context);
+  const { account_name, allow_items } = data;
+
+  return (
+    <Window title="Cryopod Console">
+      <Window.Content>
+        <Section title={`Hello, ${account_name || '[REDACTED]'}!`}>
+          This automated cryogenic freezing unit will safely store your
+          corporeal form until your next assignment.
+        </Section>
+        <CrewList />
+        {!!allow_items && <ItemList />}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const CrewList = (props, context) => {
+  const { data } = useBackend(context);
+  const { frozen_crew } = data;
+
+  return (
+    <Collapsible title="Stored Crew">
+      {!frozen_crew.length ? (
+        <NoticeBox>No stored crew!</NoticeBox>
+      ) : (
+        <Section fill scrollable>
+          <LabeledList>
+            {frozen_crew.map((person) => (
+              <LabeledList.Item key={person} label={person.name}>
+                {person.rank}
+              </LabeledList.Item>
+            ))}
+          </LabeledList>
+        </Section>
+      )}
+    </Collapsible>
+  );
+};
+
+const ItemList = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { frozen_items } = data;
+
+  const replaceItemName = (item) => {
+    let itemName = item.toString();
+    if (itemName.startsWith('the ')) {
+      itemName = itemName.slice(4, itemName.length);
+    }
+    return itemName.replace(/^\w/, (c) => c.toUpperCase());
```

Comment:
Is the `/^\w/` regex? What is it matching?

Reply:
First character in the string, any word character (a-zA-Z0-9_)

Reply:
Checks out, thanks.

Ok to be more straightforward, can you instead do an `import { toTitleCase } from 'common/string';` at the top of the file, then here do:
```js
return toTitleCase(itemName);
```